File: Práctica Grupal 1/src/gestorAplicacion/paquete1/Evento.java
package gestorAplicacion.paquete1;
import java.util.*;
import java.io.Serializable;

public class Evento implements Serializable{
	private static final long serialVersionUID = 1L;
	private Biblioteca biblioteca;
	private Sala sala;
	ArrayList<Recurso> material;
	private TipoEvento tipo;
	private static ArrayList<TipoEvento> tipos = new ArrayList<TipoEvento>();
	static {
		tipos.add(TipoEvento.CHARLA);
		tipos.add(TipoEvento.PRESENTACION);
		tipos.add(TipoEvento.ESTUDIO);
	}
	
	enum TipoEvento {
		CHARLA,
		PRESENTACION,
		ESTUDIO,
	}
	
	public Evento(byte tipo, Biblioteca sede, Sala sala) {
		this.tipo = tipos.get(tipo);
		this.biblioteca = sede;
		this.sala = sala;
		material = new ArrayList<Recurso>();
	}
	
	public Biblioteca getBiblioteca() {
		return biblioteca;
	}
	
	public ArrayList<Recurso> getMaterial() {
		return material;
	}
	
	public Sala getSala() {
		return sala;
	}
	public TipoEvento getTipo() {
		return tipo;
	}
	
	public void setBiblioteca(Biblioteca biblioteca) {
		this.biblioteca = biblioteca;
	}
	
	public void setMaterial(ArrayList<Recurso> material) {
		this.material = material;
	}
	public void setSala(Sala sala) {
		this.sala = sala;
	}
	public void setTipo(TipoEvento tipo) {
		this.tipo = tipo;
	}
	public static void setTipos(ArrayList<TipoEvento> tipos) {
		Evento.tipos = tipos;
	}
	
}

File: Práctica Grupal 2/practica2-grupo1-Equipo2/src/gestorAplicacion/clasesDeAdministracion/Evento.py
from .Biblioteca import Biblioteca
from .Sala import Sala
from .Recurso import Recurso

class Evento:
    tipos = ["CHARLA", "PRESENTACION", "ESTUDIO"]

    def __init__(self, tipo, sede, sala):
        self.tipo = Evento.tipos[tipo]
        self.biblioteca = sede
        self.sala = sala
        self.material = []

    def get_biblioteca(self):
        return self.biblioteca

    def get_material(self):
        return self.material

    def get_sala(self):
        return self.sala

    def get_tipo(self):
        return self.tipo

    def set_biblioteca(self, biblioteca):
        self.biblioteca = biblioteca

    def set_material(self, material):
        self.material = material

    def set_sala(self, sala):
        self.sala = sala

    def set_tipo(self, tipo):
        self.tipo = tipo

    @staticmethod
    def get_tipos():
        return Evento.tipos
